dotcom-rendering/src/lib/getAdPositions.test.ts

```diff
@@ -25,7 +29,7 @@ describe('Mobile Ads', () => {
 	// 3: when it's a network front and collections are equal or more than 4
 	// 2: when collections are equal or more than 4 and is not a network front
 	// 0: when collections are less than 4
-	it.each([3, 2, 0])(
+	it.each([2, 1, 0])(
```

Comment:
non-blocking: would it be worth updating the comments above to reflect the new positions?

Reply:
Good spot will do!